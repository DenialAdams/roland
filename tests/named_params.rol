proc dual_custody_lock(named param_1: bool, named param_2: u8) {
   if param_1 & (param_2 < 10) {
      print("Nuclear missiles launched");
   }
}

proc main() {
   dual_custody_lock(param_1: true, param_2: 5);
   dual_custody_lock(param_2: 3, param_1: true);
} 
