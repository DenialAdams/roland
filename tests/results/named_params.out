Nuclear missiles launched
Nuclear missiles launched
